Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -204,5 +205,12 @@ public string Comment
             get;
             set;
         }
+
+        [JsonProperty(PropertyName = "installer_command_line_params")]
+        public string InstallationArgs
```

Comment:
Just for code clarity - I'd suggest to move this property definition above the properties that are not controlled by customer like `ProvisioningState` and `InstalledDateTime` (from my understanding of the code)